<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1515</width>
    <height>925</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>1471</width>
      <height>551</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>690</y>
      <width>1471</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>跳表输出窗口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>630</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>测试结果输出窗口</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1515</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>删除</string>
    </property>
    <addaction name="actEarse"/>
    <addaction name="actEarseMin"/>
    <addaction name="actEarseMax"/>
    <addaction name="actClear"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>初始化</string>
    </property>
    <addaction name="actInitiate"/>
    <addaction name="actRandomIni"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>插入</string>
    </property>
    <addaction name="actInsertOnly"/>
    <addaction name="actInsert"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>查找</string>
    </property>
    <addaction name="actSearch"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>测试</string>
    </property>
    <addaction name="actTestStart"/>
    <addaction name="actTestStop"/>
    <addaction name="actDsize"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>清空</string>
    </property>
    <addaction name="actClearListShow"/>
    <addaction name="acClearTestShow"/>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>调整</string>
    </property>
    <addaction name="actAdjust"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_7"/>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actInitiate">
   <property name="text">
    <string>初始化</string>
   </property>
   <property name="toolTip">
    <string>初始化调表</string>
   </property>
  </action>
  <action name="actInsertOnly">
   <property name="text">
    <string>插入单个元素</string>
   </property>
   <property name="toolTip">
    <string>插入单个元素</string>
   </property>
  </action>
  <action name="actEarse">
   <property name="text">
    <string>删除指定元素</string>
   </property>
   <property name="toolTip">
    <string>删除</string>
   </property>
  </action>
  <action name="actSearch">
   <property name="text">
    <string>查找指定元素</string>
   </property>
   <property name="toolTip">
    <string>查找指定元素</string>
   </property>
  </action>
  <action name="actClear">
   <property name="text">
    <string>删除跳表</string>
   </property>
   <property name="toolTip">
    <string>删除整个跳表</string>
   </property>
  </action>
  <action name="actEarseMin">
   <property name="text">
    <string>删除最小</string>
   </property>
   <property name="toolTip">
    <string>删除关键字最小的元素</string>
   </property>
  </action>
  <action name="actEarseMax">
   <property name="text">
    <string>删除最大</string>
   </property>
   <property name="toolTip">
    <string>删除关键字最大的元素</string>
   </property>
  </action>
  <action name="actRandomIni">
   <property name="text">
    <string>随机初始化</string>
   </property>
   <property name="toolTip">
    <string>随机初始化跳表</string>
   </property>
  </action>
  <action name="actInsert">
   <property name="text">
    <string>插入多个元素</string>
   </property>
  </action>
  <action name="actTestStart">
   <property name="text">
    <string>开始时间测试</string>
   </property>
   <property name="toolTip">
    <string>开始跳表操作时间测试</string>
   </property>
  </action>
  <action name="actTestStop">
   <property name="text">
    <string>结束时间测试</string>
   </property>
   <property name="toolTip">
    <string>结束跳表操作时间测试</string>
   </property>
  </action>
  <action name="actClearListShow">
   <property name="text">
    <string>清空跳表显示框</string>
   </property>
   <property name="toolTip">
    <string>清空跳表显示框</string>
   </property>
  </action>
  <action name="acClearTestShow">
   <property name="text">
    <string>清空测试数据显示框</string>
   </property>
   <property name="toolTip">
    <string>清空测试数据显示框</string>
   </property>
  </action>
  <action name="actAdjust">
   <property name="text">
    <string>重新调整跳表</string>
   </property>
   <property name="toolTip">
    <string>重新严格初始化跳表</string>
   </property>
  </action>
  <action name="actDsize">
   <property name="text">
    <string>节点数量</string>
   </property>
   <property name="toolTip">
    <string>查询节点数量</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>acClearTestShow</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>670</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actClearListShow</sender>
   <signal>triggered()</signal>
   <receiver>plainTextEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>670</x>
     <y>465</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
